src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -645,6 +645,10 @@
           type: int
           short-summary: NAT gateway idle timeout in minutes.
           long-summary: Desired idle timeout for NAT gateway outbound flows, default is 4 minutes. Please specify a value in the range of [4, 120]. Valid for Standard SKU load balancer cluster with managedNATGateway outbound type only.
+        - name: --outbound-type
+          type: string
+          short-summary: How outbound traffic will be configured for a cluster.
```

Comment:
This description seems a bit confusing. Could you explain the specific feature of this parameter directly and concisely?

Reply:
This will option will change the way how the outbound connections are managed in a aks cluster, Default is loadbalancer, other available options are managedNATGateway, userassignedNATGateway, UDR